src/servers/trigger.ts

```diff
@@ -181,15 +214,18 @@ const listenerOnchain = ({ lnd }) => {
   subTransactions.on("chain_transaction", onchainTransactionEventHandler)
 
   subTransactions.on("error", (err) => {
-    baseLogger.info({ err }, "error subTransactions")
+    baseLogger.error({ err }, "error subTransactions")
     subTransactions.removeAllListeners()
```

Comment:
maybe the comment about removing listen also apply here

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/428/commits/20668101e2a16ba4ad949eaa1af8980a7bbc26ed